libs/api/src/lib/memberContext.ts

```diff
@@ -273,10 +273,14 @@ export class MemberContext implements MemberContextInterface {
         days: subscriptionFlowActionDeactivationUnpaid.daysAwayFromEnding
       })
 
+      const memberplan = await this.prisma.memberPlan.findUnique({
+        where: {id: subscription.memberPlanID}
+      })
+
       const newInvoice = await this.prisma.invoice.create({
         data: {
           subscriptionID: subscription.id,
-          description: `Membership from ${startDate.toISOString()} for ${user.name || user.email}`,
+          description: `${memberplan.name}: ${format(startDate, 'dd-MM-yyyy')}`,
```

Comment:
I see that the user's email or name are no longer present here. Aren't they useful for the financial accounting at the media? Or do they see them anyway?